### DISCUSSION TITLE
v2.3.0

### DISCUSSION THREAD

**Discussion by cdubz** [babybuddy/babybuddy#796] [General]
## What's Changed
* feat: create a trimmed form just for bottle feeding. by @billybonks in https://github.com/babybuddy/babybuddy/pull/756
* ci: add djlint to ci by @billybonks in https://github.com/babybuddy/babybuddy/pull/763
* feat: add links for bottle feeding by @billybonks in https://github.com/babybuddy/babybuddy/pull/764
* bugfix: dashboard shows ago when less then 1minute by @billybonks in https://github.com/babybuddy/babybuddy/pull/768
* Add prettier by @billybonks in https://github.com/babybuddy/babybuddy/pull/762
* Feat: Increase mobile friendliness of forms by @billybonks in https://github.com/babybuddy/babybuddy/pull/760
* Add external developer documentation to BBoTW README.md by @barakplasma in https://github.com/babybuddy/babybuddy/pull/776
* deploy: rebuild static files for mobile friendly forms by @billybonks in https://github.com/babybuddy/babybuddy/pull/778
* Add feeding method history by @abog854 in https://github.com/babybuddy/babybuddy/pull/774
* Add shortcuts to manifest by @ahosgood in https://github.com/babybuddy/babybuddy/pull/783
* Add Ukrainian translations by @cdubz in https://github.com/babybuddy/babybuddy/pull/786
* Add Hebrew translations by @cdubz in https://github.com/babybuddy/babybuddy/pull/787
* Exclude /api from PROXY forward auth by @zjean in https://github.com/babybuddy/babybuddy/pull/790

## New Contributors
* @billybonks made their first contribution in https://github.com/babybuddy/babybuddy/pull/756
* @abog854 made their first contribution in https://github.com/babybuddy/babybuddy/pull/774
* @ahosgood made their first contribution in https://github.com/babybuddy/babybuddy/pull/783
* @zjean made their first contribution in https://github.com/babybuddy/babybuddy/pull/790

**Full Changelog**: https://github.com/babybuddy/babybuddy/compare/v2.2.1...v2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/babybuddy/babybuddy/releases/tag/v2.3.0'>v2.3.0</a>.</em>

**Comment by larsreinhardt** [comment]
@cdubz did the amount value Field in diaper change get lost in this release? Because i cannot input it

**Reply by cdubz** [reply]
Ack. Yes. We overlooked a few fields there while reworking for the mobile form. See also #799. Will get it back in a bug fix release.
